scripts/release/mule/test/test.sh

```diff
@@ -16,9 +16,15 @@ export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
 export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
 export SHA=${SHA:-$(git rev-parse HEAD)}
 export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || echo algorand)
```

Comment:
Maybe expand this from one liners, and add algorand-nightly support?

Reply:
One-liners rule!

Are we officially supporting nightlies now?  I thought that was still being determined.

Reply:
Hmm, maybe not for packaging. Something to look at in the future.